source/org/zfin/infrastructure/EnumValidationService.java

```diff
@@ -170,7 +170,9 @@ public void validateCurationRole() throws EnumValidationException {
     public void validateCurationLocation() throws EnumValidationException {
         String hql = "select distinct ptl_location_display from pub_tracking_location";
         List locationList = HibernateUtil.currentSession().createNativeQuery(hql).list();
```

Comment:
Would it be good to add a type to this, like `createNativeQuery(hql, String.class)...` ?

Reply:
Good idea.

Reply:
Changed it also in the other methods.